tracing-subscriber/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 0.2.5 (April 21, 2020)
+
+### Changed
+
+- **fmt**: Bump sharded-slab dependency (#679)
```

Comment:
dunno if this _really_ needs to go in the changelog; your call

Reply:
would prefer to be slightly noisier. It's valuable to Adam.